### ISSUE TITLE
Paper 191 time out

### ISSUE THREAD

**Issue by crazykidney** [PaperMC/Paper#4312]
### Link/paste of stack trace
https://pastebin.com/raw/WdW39Y6J

### Plugin list:
custom

### Paper version:
191

What happened?

**Comment by Chew** [comment]
1) Please fill out the entire template.
2) We need a lot more than just a tiny stack trace, but from what I assume it likely says DO NOT REPORT TO PAPER. You probably just have a lot of lag.
3) Join our [discord server](https://discord.gg/papermc) and we will be happy to help diagnose lag issues there.

**Comment by zachthedev** [comment]
I am having this problem. Here are my details (instead of creating a new issue)

**Link/paste of stack trace**
This is from Server 2 (server 1 hasn't thrown a thread dump yet)
```
[14:19:45 WARN]: Can't keep up! Is the server overloaded? Running 13041ms or 260 ticks behind
[14:19:45 ERROR]: --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH  - git-Paper-191 (MC: 1.16.3) ---
[14:19:45 ERROR]: The server has not responded for -3 seconds! Creating thread dump
[14:19:45 ERROR]: ------------------------------
[14:19:45 ERROR]: Server thread dump (Look for plugins here before reporting to Paper!):
[14:19:45 ERROR]: ------------------------------
[14:19:45 ERROR]: Current Thread: Server thread
[14:19:45 ERROR]:       PID: 29 | Suspended: false | Native: false | State: RUNNABLE
[14:19:45 ERROR]:       Stack:
[14:19:45 ERROR]:               java.lang.Thread.yield(Native Method)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.IAsyncTaskHandler.bl(IAsyncTaskHandler.java:130)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:120)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.MinecraftServer.a(MinecraftServer.java:1202)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:999)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177)
[14:19:45 ERROR]:               net.minecraft.server.v1_16_R2.MinecraftServer$$Lambda$3224/828440307.run(Unknown Source)
[14:19:45 ERROR]:               java.lang.Thread.run(Thread.java:748)
[14:19:45 ERROR]: ------------------------------
[14:19:45 ERROR]: --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH ---
[14:19:45 ERROR]: ------------------------------

```
(I know it says not to report to you, but this seems like something from paper since another person is having similar issues)

**Plugin list:**
```
Server 1: 
AFKPlus (3.2.0), AFKPlusPrefix (1.1.1), ChatItemDisplay (1.5.6), DriveBackupV2 (1.3.5), HexNicks (1.0), LuckPerms (5.1.26), OpenInv (4.1.4), PlugMan (2.1.7), ProtocolLib (4.6.0-SNAPSHOT-b478), ServerListPlus (3.5.0-SNAPSHOT), SpectatorMode (1.3.14), SuperVanish (6.2.0), TabList (5.4.6), ViaBackwards (3.2.0-SNAPSHOT), ViaVersion (3.2.0-SNAPSHOT), WorldEdit (7.1.0;8e55131)
```
```
Server 2: 
AsyncWorldEdit (3.8.0), AsyncWorldEditBossBar (1.1.1), DriveBackupV2 (1.3.5), LuckPerms (5.1.26), PlugMan (2.1.7), ProtocolLib (4.6.0-SNAPSHOT-b476), TabList (5.4.5), WorldEdit (7.2.0-beta-04+98f71a3), WorldSpawn (1.4.4)
```

**Paper version:**
191

What happened?
Very intermittent, but occasionally the server will overload (about 100-400 ticks). I just had server 2 throw a thread dumb.
The only things I had changed were ProtocoLib (moving over to the dev build to get 1.16.3) and Paper 191.

Unable to reproduce it, it just happens. This is without any player involvement. All I did was spin up the server and let it sit.

**Comment by Chew** [comment]
We need some timings reports to reliably figure out what's causing the issue.

**Comment by zachthedev** [comment]
@Chew 
[This](https://timings.aikar.co/?id=f0dbcf31d37645cd805bbf919ca94e40) is my timings report on Server 1. 
It looks like it only starts to happen intermittently after a player logs in. I started up server 1, joined it, then left. Then shortly after it started to spew a couple 'is the server overloaded' warnings.
EDIT:
[Here](https://timings.aikar.co/?id=a7680d437641426da13111b78089e2e0) are the timings from Server 2. Same thing. I logged in and then logged out. Then about 30 minutes later, it threw the 'is the server overloaded' warning.
